src/index.js

```diff
@@ -76,4 +33,22 @@ function init() {
   getGrid().append(renderedDays);
 }
 
-init();
+function clickHandler(event) {
+  if(event.target.className.split(' ').includes('day')) {
```

Comment:
className can be string with different elements, like `day highlighted`, to we need to convert it to array